src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/index.tsx

```diff
@@ -78,7 +93,7 @@ const ContractInteraction = ({ contractAddress, initialValues, onClose, onNext }
         formMutators={formMutators}
         initialValues={initialValues}
         onSubmit={handleSubmit}
-        subscription={{ submitting: true, pristine: true }}
+        subscription={{ submitting: true, pristine: true, values: true }}
```

Comment:
here you track this with final-form, is it possible to do the same in `src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/SendCustomTx/index.tsx`?